src/pages/workspace/categories/WorkspaceCategoriesPage.tsx

```diff
@@ -50,6 +49,7 @@ function WorkspaceCategoriesPage({policyCategories, route}: WorkspaceCategoriesP
     const theme = useTheme();
     const {translate} = useLocalize();
     const [selectedCategories, setSelectedCategories] = useState<Record<string, boolean>>({});
+    const dropdownButtonRef = useRef(null);
```

Comment:
The ref must be typed.
```suggestion
    const dropdownButtonRef = useRef<ButtonWithDropdownMenu | null>(null);
```